dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops

```diff
@@ -820,3 +820,12 @@ CWWKD1081.entity.general.err.explanation=An error prevents the \
  specified entity from being used.
 CWWKD1081.entity.general.err.useraction=Locate the entity and make \
  corrections if needed.
+
+CWWKD1082.entity.classes.missing=CWWKD1082E: The persistence unit for the \
+ {0} data store does not define one or more entity classes ({1}) \
+ that are required by one or more repository interfaces ({2}) \
+ that use the data store.
+CWWKD1082.entity.classes.missing.explanation=All entity classes that are \
```

Comment:
Repositories use entity classes that must be defined in the persistence unit.

Reply:
Updating as suggested.